terra_notebook_utils/drs.py

```diff
@@ -159,19 +159,11 @@ def get_drs_blob(drs_url_or_info: Union[str, DRSInfo],
         raise TypeError()
     return GSBlob(info.bucket_name, info.key, info.credentials, workspace_namespace)
 
-def resolve_drs_for_gs_storage(drs_url: str) -> Tuple[gs.Client, DRSInfo]:
-    """Attempt to resolve gs:// url and credentials for a DRS object. Instantiate and return the Google Storage
-    client.
-    """
-    info = get_drs_info(drs_url)
-
-    if info.credentials is not None:
-        project_id = info.credentials['project_id']
+def blob_for_url(url: str, workspace_namespace: Optional[str]=WORKSPACE_GOOGLE_PROJECT) -> Blob:
+    if url.startswith("drs://"):
+        return get_drs_blob(url, workspace_namespace)
```

Comment:
Hmmm... could `copy_client.blob_for_url(url)` also just work for DRS URIs?  Not a strong suggestion.

Reply:
That would make blobstore and drs interdependent, so not easily.